third_party/proton/test/instrument.py

```diff
@@ -0,0 +1,79 @@
+import torch
+
+import triton
+import triton.language as tl
+
+
+@triton.autotune(
+    configs=[
+        triton.Config({
+            "BLOCK_SIZE_M": 128,
+            "BLOCK_SIZE_N": 256,
+            "BLOCK_SIZE_K": 64,
+            "GROUP_SIZE_M": 8,
+        }, ),
+    ],
+    key=["M", "N", "K"],
+)
+@triton.jit
+def matmul_kernel(a_ptr, b_ptr, c_ptr, M, N, K, stride_am, stride_ak,  #
```

Comment:
Simplify the function with the following changes:

1. Remove `triton.autotune`.
2. Reduce the length of the triton function.